tasks/installApache.yml

```diff
@@ -12,11 +12,12 @@
   become: True
   package:
     name:
-      - "{{ ( ansible_os_family == 'Debian' ) | ternary( 'policycoreutils-python-utils','policycoreutils-python' ) }}"
+      - policycoreutils-python
       - selinux-policy
     state: present
   register: installed_packages
   until: installed_packages is succeeded
+  when: ansible_os_family == 'RedHat'
```

Comment:
Why using such conditionals instead of including related tasks for different OS versions?

Reply:
replaced